app/controllers/miq_ae_class_controller.rb

```diff
@@ -1499,8 +1499,12 @@ def create_ns
       @in_a_form = false
       replace_right_cell
     when "add"
-      parent_id = @edit[:typ] == "MiqAeDomain" ? nil : from_cid(x_node.split('-')[1])
-      add_ae_ns = @edit[:typ].constantize.new(:parent_id => parent_id)
+      parent_id, tenant = if @edit[:typ] == "MiqAeDomain"
```

Comment:
@kbrock 
The Tenant need to be set only for Domains and not namespaces. They are both stored in the same table miq_ae_namespace.